test/kubernetes_unittest.cc

```diff
@@ -717,41 +717,72 @@ bool WaitForNewerCollectionTimestamp(const MetadataStore& store,
   return false;
 }
 
-TEST_F(KubernetesTest, KubernetesUpdater) {
+void TestKubernetesUpdater(bool cluster_level_metadata, bool service_metadata) {
+  const std::string nodes_path = cluster_level_metadata ?
```

Comment:
I was confused at how the SetResponse and AllowStream node paths were different. Could we suffix these with _watch_path?

Reply:
Done.